android/src/main/java/com/reactnativecommunity/webview/RNCWebViewManager.java

```diff
@@ -413,6 +414,10 @@ public boolean onConsoleMessage(ConsoleMessage message) {
       public void onGeolocationPermissionsShowPrompt(String origin, GeolocationPermissions.Callback callback) {
         callback.invoke(origin, true, false);
       }
+
+      public boolean onShowFileChooser(WebView webView, ValueCallback<Uri[]> filePathCallback, FileChooserParams fileChooserParams) {
```

Comment:
This does not gets triggered below 5.0

There are others function signature that allow unofficial support but I'm not sure there is a point to support this below 5.0

Reply:
4.1-4.4 (SDK 16-19) which means JellyBean & KitKat cover more than 10% of market share.
If it's easy to cover that, I think it's definitely worth it.
I would like to give it a shot, at least.